doc/wintabs.txt

```diff
@@ -200,6 +200,32 @@ default: ' '
                 '[]' and ')(' are some of the good ones. It's recommended that
                 they have the same length as separators.
 
+*g:wintabs_ui_show_vimtab_name*
+values: 0, 1 or 2
+default: 0
+                This controls how Vim tabs will be displayed.
+                0: show a number of the tab.
+                1: show a name of the tab.
+                2: show number and name of the tab. (<number>:<tab_name>).
```

Comment:
Yes, but it can be useful for someone who likes navigation as in tmux. For example, in tmux my tabs  is named like <tab_number>:<tab_name>. I enabled mouse mode in my tmux config but mostly I'm changing tabs using the number of the tab. It is more comfortable for me.